docs/user-guide/02_physics/01_plasma.md

```diff
@@ -24,9 +24,7 @@ text-align: center;
 ![](|media|/cross_section.png){: style="float: left"}
 It is common in Tokamaks to input the plasma parameters and fields as flux functions.
 However, this approach makes several assumptions that may not be true in non-tokamak devices.
-In FIDASIM the plasma parameters and fields are mapped onto a 2D R-Z grid.
-This only assumes axisymmetric profiles within the region of interest.
-In addition to plasma parameters a boolean mask is used to indicate where the plasma and fields are well defined.
+In FIDASIM, the plasma parameters and fields are mapped onto a 3D cylindrical grid, and a boolean mask is used to indicate where the plasma and fields are well defined.
```

Comment:
The plasma parameters and fields are only mapped onto the 3D grid if a phi variable is present. You need to mention this.